frontend/src/Components/AppointmentsDayView/__tests__/View.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
I like it!

Maybe tomorrow we will have time to write some tests on the other views of the application.